docs/content/3.manual/manual.yml

```diff
@@ -271,17 +271,24 @@ sections:
             input: '"Hello, world!"'
             output: ['"Hello, world!"']
 
-      - title: "`.foo`, `.foo.bar`"
+      - title: "Object Identifier-Index: `.foo`, `.foo.bar`"
```

Comment:
JSON speaks of `objects` with `name`/`value` pairs.  jq calls them `objects` with `key`/`value` pairs.  There will be no talk of fields or members.